Makefile

```diff
@@ -18,16 +19,31 @@ all: install build
 .PHONY: all
 
 install:
+ifeq ($(OS),Windows_NT) 
+	setup_env.bat
+else
 	./setup_env.sh
+endif
+
 .PHONY: install
 
 genproto:
+ifeq ($(OS),Windows_NT) 
+	scripts\win\genproto.bat
+else
 	./scripts/genproto.sh
+endif
+
 .PHONY: genproto
 
 build:
+ifeq ($(OS),Windows_NT)
```

Comment:
getting this error

make genproto
make[1]: Entering directory `C:/Users/User/go-spacemesh'
scripts\win\genproto.bat
Generating protobuf for C:\Users\User\go-spacemesh\api\pb\api.proto
The system cannot find the path specified.
Generating protobuf for C:\Users\User\go-spacemesh\consensus\pb\messages.proto
The system cannot find the path specified.
Generating protobuf for C:\Users\User\go-spacemesh\hare\pb\hare.proto
The system cannot find the path specified.
Generating protobuf for C:\Users\User\go-spacemesh\merkle\pb\node.proto
The system cannot find the path specified.
Generating protobuf for C:\Users\User\go-spacemesh\p2p\dht\pb\findnode.proto
The system cannot find the path specified.
Generating protobuf for C:\Users\User\go-spacemesh\p2p\pb\message.proto
The system cannot find the path specified.
Generating protobuf for C:\Users\User\go-spacemesh\ping\pb\ping.proto
The system cannot find the path specified.
Generating protobuf for C:\Users\User\go-spacemesh\sync\pb\sync.proto
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
make[1]: Leaving directory `C:/Users/User/go-spacemesh'
go build -ldflags "-X main.version=0.0.1 -X main.commit=f8d87ac48afd6aa2de24365f163776eda979e282 -X main.branch=develop" -o C:\Users\User\go-spacemesh/go-spacemesh
go: finding github.com/spacemeshos/go-spacemesh/p2p/pb latest
go: finding github.com/spacemeshos/go-spacemesh/p2p/dht/pb latest
go: finding github.com/spacemeshos/go-spacemesh/api/pb latest
go: finding github.com/spacemeshos/go-spacemesh/p2p latest
go: finding github.com/spacemeshos/go-spacemesh/p2p/dht latest
go: finding github.com/spacemeshos/go-spacemesh/api latest
api\grpc_server.go:6:2: unknown import path "github.com/spacemeshos/go-spacemesh/api/pb": cannot find module providing package github.com/spacemeshos/go-spacemesh/api/pb
p2p\dht\findnode.go:9:2: unknown import path "github.com/spacemeshos/go-spacemesh/p2p/dht/pb": cannot find module providing package github.com/spacemeshos/go-spacemesh/p2p/dht/pb
p2p\net\handshake.go:19:2: unknown import path "github.com/spacemeshos/go-spacemesh/p2p/pb": cannot find module providing package github.com/spacemeshos/go-spacemesh/p2p/pb
make: *** [build] Error 1